devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -41,14 +42,17 @@ metadata {
 	}
 
 	tiles(scale: 2) {
-		multiAttributeTile(name:"windowShade", type: "generic", width: 6, height: 4) {
+		multiAttributeTile(name:"windowShade", type: "lighting", width: 6, height: 4) {
```

Comment:
not that it matters since this is classic, but doesn't this put in a lightbulb icon?

Reply:
It should use the icons defined in the `attributeState` definitions.